api/utils/points.ts

```diff
@@ -0,0 +1,15 @@
+import { getDbClient } from "./database";
+export async function addPoints(userEmail: string, type: number) {
```

Comment:
Let the variable here be points
```suggestion
export async function addPoints(userEmail: string, points: number) {
```